.github/workflows/book.yml

```diff
@@ -38,6 +38,8 @@ jobs:
 
       - name: Setup mdBook
         uses: peaceiris/actions-mdbook@v1
+        with:
+          mdbook-version: "latest"
```

Comment:
latest is the default no?

Reply:
Mmmh, but I used the latest on Gitlab as well...

Then that might not be it.

Reply:
I also tried it locally, and there this problem does not exist.

Maybe install mdbook with cargo for now?

Reply:
Or like the book does
https://github.com/rust-lang/book/blob/master/.github/workflows/main.yml

Reply:
This can be dropped per the matrix conversation :-)